### ISSUE TITLE
Server can't open

### ISSUE THREAD

**Issue by Louuuup** [simibubi/Above-and-Beyond#474]
Hi! I've downloader the server file from the curseforge launcher, but encoutered this problem. When I execute the server .jar or the .bat file I made, the program launches and instantly crashes within half a second. In the lastest log, I am able to find this line: 
`[main/ERROR] [mixin/]: Mixin config antiqueatlas.mixins.json does not specify "minVersion" property`
What should I do in that case? The forge version is 1.16.5-36.2.20, same as the client I have. Haven't modified anything, and am not sure of what to do next. 

Any ideas?

**Comment by mmedwards44** [comment]
Also having this issue. SS attached of my terminal.
![createServerIssue](https://user-images.githubusercontent.com/95927583/187054674-a08b011c-dbc8-41a6-be34-118825dc96c2.JPG)

**Comment by ghost** [comment]
https://github.com/simibubi/Above-and-Beyond/issues/482 This is your real issue.

**Comment by IzdebskiMichal** [comment]
Had this issue as well:
![2022-09-19_10h50_36](https://user-images.githubusercontent.com/36077834/190984165-1c9082fc-8ac3-4099-8d9c-f19fe773d551.png)


I could not find the answer anywhere so I tried to uninstall my latest version of Java (Java 8 update 341) and then download any other previous update of Java 8. I picked update 292 and it actually worked. I'm pretty surprised that there isn't any update on this issue.

I got the previous java from this link: https://www.openlogic.com/openjdk-downloads

I hope I helped someone :)
